tutorials/bios-boot-tutorial/README.md

```diff
@@ -7,38 +7,69 @@ The `bios-boot-tutorial.py` script simulates the EOSIO bios boot sequence.
 1. Python 3.x
 2. CMake
 3. git
+4. g++
+5. build-essentials
+6. pip3
+7. openssl
+8. curl
+9. jq
+10. psmisc
+
 
 ``Steps``:
 
 1. Install eosio binaries by following the steps outlined in below tutorial
-[Install eosio binaries](https://github.com/EOSIO/eos#mac-os-x-brew-install)
+[Install eosio binaries](https://github.com/EOSIO/eos/tree/release/2.0.x#mac-os-x-brew-install).
+
+2. Install eosio.cdt version 1.6.3 binaries by following the steps outlined in below tutorial
+[Install eosio.cdt binaries](https://github.com/EOSIO/eosio.cdt/tree/release/1.6.x#binary-releases).
+
+3. Compile `eosio.contracts` version 1.8.x.
+
+```bash
+$ cd ~
+$ git clone https://github.com/EOSIO/eosio.contracts.git eosio.contracts-1.8.x
+$ cd ./eosio.contracts-1.8.x/
+$ git checkout release/1.8.x
+$ ./build.sh
+$ cd ./build/contracts/
+$ pwd
+
+```
+
+4. Make note of the directory where the contracts were compiled. 
+The last command in the previous step printed on the bash console the contracts' directory, make note of it, we'll reference it from now on as `EOSIO_OLD_CONTRACTS_DIRECTORY`.
```

Comment:
`EOSIO_OLD_CONTRACTS_DIRECTORY` would not longer be an environment variable to be used in this tutorial.

Instead you could introduce the environment variable `EOSIO_BOOT_CONTRACT` which the user sets to the `contracts/eosio.bios/` directory in the build folder within which they compiled v1.8.x eosio.contracts in sub-steps 2 to 4 above. If the user chose not to follow these optional sub-steps, they would set the `EOSIO_BOOT_CONTRACT` environment variable to the `tutorials/bios-boot-tutorial/eosio.bios` path within the EOSIO source directory.

Reply:
@iamveritas ☝️

Reply:
I will resolve the rest of the comments because they all stem from the initial proposal which we will not go with.